javascript/javascript1/week4/classExercises/characterFrequencies.js

```diff
@@ -0,0 +1,21 @@
+
+function getCharacterFrequencies(str) {
+  const strWithoutSpace = str.trim().replace(" ", "");
+  const letters = strWithoutSpace.split("");
+  const frequencies = {};
+  const objToPrint = { characters: [], length: strWithoutSpace.length };
+
+  for (let char of letters) {
+    if (frequencies[char]) {
+      frequencies[char]++;
+    } else {
+      frequencies[char] = 1;
+    }
```

Comment:
You might see it sooner or later, but here is how to write `if else` statements in short-hand:
```suggestion
    frequencies[char] ? frequencies[char]++ : frequencies[char] = 1;
```

First is the condition that needs to return true, followed by a question mark, then the code executed in this case and the one in `else` case, are separated with a column `:`.